.pre-commit-config.yaml

```
@@ -1,11 +1,11 @@
 repos:
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.241
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.4.2
     hooks:
       - id: ruff
 
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: 24.4.2
```

```suggestion
    rev: 23.1.0
```

Go back to the version that accepts the code as is - we can update the style in another PR.